### DISCUSSION TITLE
Some Questions

### DISCUSSION THREAD

**Discussion by ghost** [The-Run-Philosophy-Organization/run#150] [General]
1.我国媒体的内宣经常会翻车（笑），这部分能写到此项目中吗？
这应该算“茶余饭后，博君一笑”的部分（

2.众所周知，我国有着墙大的言论审查制度，此部分能否添加到润学的哲学概念部分呢？

**Comment by the-run-philosophy** [comment]
关于第二点，欢迎详尽解释言论审查制度的机制，原理和例子。谢谢。
这是千秋万代的功业

**Comment by the-run-philosophy** [comment]
关于第一点，欢迎提交PR
